.github/workflows/ci.yml

```diff
@@ -1,91 +1,32 @@
 name: CI
 
 on:
-  push:
   pull_request:
-  schedule: # Trigger a job on master at midnight UTC every day
-    - cron: '0 0 * * *'
```

Comment:
You wan to keep these lines here. They:
1. run pipelines on every pushed commit
2. run pipelines on every pr (iirc they run on a merged reference but dont quote me on that)
3. run pipelines on the master every day at midnight to make sure that 3rd party package updates don't introduce breaking changes for us

Reply:
we probs actually want to change 3 to run on both master and ros2 for now